### PR TITLE
Update initial_sdk_setup.md

### PR DESCRIPTION
**Pull request by davidbielik**
more words

### CODE REVIEW

File: _docs/_developer_guide/platform_integration_guides/unreal_engine/initial_sdk_setup.md [hunk 1]
```diff
@@ -6,11 +6,22 @@ page_order: 0
 
 # Unreal Engine SDK
 
-The Braze Unreal Engine SDK integrates the Braze SDK for Android and iOS with UE4.
+The Braze Unreal Engine plugin lets you integrate with Unreal Engine, a state-of-the-art real-time engine and editor.
 
-Please see our [technical documentation][1] to get started.
+With Braze, you can now measure and engage with your users on your iOS and Android apps built using Unreal Engine 4.
```
Comment (davidbielik) [inline _docs/_developer_guide/platform_integration_guides/unreal_engine/initial_sdk_setup.md:11]:
```suggestion
With Braze, you can now measure and engage with users on your iOS and Android apps built using Unreal Engine 4.
```

File: _docs/_developer_guide/platform_integration_guides/unreal_engine/initial_sdk_setup.md [hunk 2]
```diff
@@ -6,11 +6,22 @@ page_order: 0
 
 # Unreal Engine SDK
 
-The Braze Unreal Engine SDK integrates the Braze SDK for Android and iOS with UE4.
+The Braze Unreal Engine plugin lets you integrate with Unreal Engine, a state-of-the-art real-time engine and editor.
 
-Please see our [technical documentation][1] to get started.
+With Braze, you can now measure and engage with users on your iOS and Android apps built using Unreal Engine 4.
```
Comment (davidbielik) [inline _docs/_developer_guide/platform_integration_guides/unreal_engine/initial_sdk_setup.md:11]:
```suggestion
With Braze, you can now measure and engage with users on your iOS and Android apps built on Unreal Engine 4.
```

File: _docs/_developer_guide/platform_integration_guides/unreal_engine/initial_sdk_setup.md [hunk 3]
```diff
@@ -6,11 +6,22 @@ page_order: 0
...
+With Braze, you can now measure and engage with users on your iOS and Android apps built on Unreal Engine 4.
+
+Features include:
+
+* Measure and track sessions within your app or game
+* Track in-app purchases and custom events
+* Update user profiles with standard and custom attributes
+* Send Push Notifications
+* Integrate your Unreal apps with larger Canvas journeys
+* Send cross-channel messaging, like email or SMS, based on in-game behaviors
```
Comment (davidbielik) [inline _docs/_developer_guide/platform_integration_guides/unreal_engine/initial_sdk_setup.md:20]:
```suggestion
* Send cross-channel messaging, like email or SMS, based on in-game behavior
```

**Review by timothy-kim — APPROVED** [review]
:

**Pull request merged**